english/src/README.md

```diff
@@ -1,18 +1,32 @@
 # @lnp2pBot
 
-On this site you will find information about [@lnp2pBot](https://t.me/lnp2pbot). It is an application to buy and sell bitcoin on Lightning Network with your local currency, without giving your personal data.
+Here you'll find information about [@lnp2pBot](https://t.me/lnp2pbot). This is a Telegram bot for buying and selling bitcoin on the Lightning Network, in a peer-to-peer manner, using your local currency without requiring KYC or providing personal data.
 
 ## Quick Start Guide
 
-1. Download a Lightning Network compatible wallet ([Recomended wallets](./recommended-wallets.md)).
-2. Get a Telegram username.
-3. Start [@lnp2pBot](https://t.me/lnp2pbot)
+1. Download a Lightning Network-compatible wallet ([Recommended Wallets](./recommended-wallets.md)).
+2. Get a Telegram user.
+3. Start [@lnp2pBot](https://t.me/lnp2pbot).
 
     ![Starting the bot](./assets/images/bot-start.gif)
 
-4. Find offers in the channel [@lnp2pbot Exchange](https://t.me/p2plightning).
+4. Find offers to buy or sell Sats at:
+- [https://market.lnp2pbot.com](https://market.lnp2pbot.com)
+- The general offer channel: [@lnp2pbot Exchange](https://t.me/p2plightning)
+- [A community that uses your fiat currency](./where-i-can-find-offers-for-my-local-currency.md)
 
-    ![Looking for offers](./assets/images/scan-channel.gif)
+    ![Searching for offers](./assets/images/scan-channel.gif)
+    
+5. When you take an offer, [@lnp2pBot](https://t.me/lnp2pbot) will guide you step by step through the buying/selling process.
+6. If you don't find a suitable offer, create one and wait for someone to take it.
+Type `/buy` in [@lnp2pBot](https://t.me/lnp2pbot) to [buy Sats](./how-do-i-create-a-buy-order.md) or `/sell` to [sell Sats](./how-do-i-create-a-sell-order.md), according to your preference, and it will guide you through the process. If you want your offer to be published in a specific community rather than the general offer channel, you need to [set a default community](./how-do-i-make-offers-in-a-community.md).
 
-5. Start buying and selling.
-6. Find answers to your questions in the [FAQ](./faq.md) section.
+7. Start buying and selling.
+8. Find answers to your questions in the [FAQ section](./faq.md).
+
+## Recommendations
+- If you have any unanswered questions at [https://lnp2pbot/aprende](https://lnp2pbot.com/aprende/), join the [support group](https://t.me/lnp2pbotHelp) and ask there.
```

Comment:
Here we should link the english documentation site https://lnp2pbot.com/learn instead

Reply:
done